src/converters/sentencetolabels/labelrecovery.py

```diff
@@ -30,8 +31,11 @@ def label_exceptive_clauses(sentence: str, labels: list[Label]) -> list[SubLabel
         exceptive_instances += [m.span() for m in iter]
 
     for index, instance in enumerate(exceptive_instances):
-        label = get_label_by_type_and_position(labels, type='Negation', begin=instance[0], end=instance[1])
-        if label == None:
-            additional_labels.append(SubLabel(id=f'AEX{index}', name='Negation', begin=instance[0], end=instance[1]))
+        label = get_label_by_type_and_position(labels, type=consts.NEGATION, begin=instance[0], end=instance[1])
+        if label is None:
+            additional_labels.append(
+                SubLabel(id=f'AEX{index}', name=consts.NEGATION, begin=instance[0], end=instance[1])
```

Comment:
Are label IDs already part of the `src.util.constants` file? If so, then `AEX` might be a relevant addition.

Reply:
I didn't touch `AEX` since I don't understand its purpose. 
If it is a general concept that needs to be present in different files, then we can also move it to `constants`, otherwise, a local const should be created. 
But I'm not what a good name for the const would be.

Reply:
I will have to investigate the usefulness of the `AEX` label myself. I have opened a new issue for this (#80), and we can close the current PR with this.